legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/result/RealizedRelationalResult.java

```diff
@@ -40,6 +40,7 @@ public class RealizedRelationalResult extends StreamingResult
     public List<List<Object>> transformedRows;
 
     private static final int DEFAULT_ROW_LIMIT = 1000;
+    private static final String ROW_LIMIT_PROPERTY_NAME = "DEFAULT_ROW_LIMIT";
```

Comment:
lets give this a name that reflect the usage and avoid colliding with other property names.  Something like org.finos.legend.engine.realizedRelationalResultRowLimit.